spring-cloud-dataflow-core/pom.xml

```diff
@@ -54,5 +54,10 @@
 			<groupId>org.springframework.hateoas</groupId>
 			<artifactId>spring-hateoas</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
I think it got in there by mistake somehow early on.  I suspect this is why I had to add all that autoconfigure excludes.... testing now with it removed.

Reply:
ah, it is there because of the update to `Launcher`, it now has a Scheduler, and I guess intellij picked the cf one, will try with the spi dependency.

Reply:
I replaced with 
```		<dependency>
			<groupId>org.springframework.cloud</groupId>
			<artifactId>spring-cloud-scheduler-spi</artifactId>
			<version>${spring-cloud-scheduler-spi.version}</version>
		</dependency>```
and things look good so far